code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -841,6 +841,26 @@
 	taste_description = "rancid fungus"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
+/datum/reagent/consumable/moltobeso
+	name = "Molt'Obeso" //pardon my Italian
+	description = "Concentrated gluttony."
+	color = "#f8fc36"
+	taste_description = "gluttony"
```

Comment:
I think this should have a low taste multiplier to make it harder to notice.

Reply:
I didn't know about the taste_mult. Made it 0.4, dunno if it's too much or too little